ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/DataStructureUtil.java

```diff
@@ -276,13 +278,36 @@ public static Exit randomExit(int seed) {
     return new Exit(randomUnsignedLong(seed), randomUnsignedLong(seed++), BLSSignature.random());
   }
 
+  public static Transfer randomTransfer() {
+    return new Transfer(
+        randomUnsignedLong(),
+        randomUnsignedLong(),
+        randomUnsignedLong(),
+        randomUnsignedLong(),
+        randomUnsignedLong(),
+        BLSPublicKey.randomPublicKey(),
+        BLSSignature.random());
+  }
+
+  public static Transfer randomTransfer(int seed) {
+    return new Transfer(
+        randomUnsignedLong(seed),
+        randomUnsignedLong(seed + 1),
+        randomUnsignedLong(seed + 2),
+        randomUnsignedLong(seed + 3),
+        randomUnsignedLong(seed + 4),
+        BLSPublicKey.randomPublicKey(),
+        BLSSignature.random());
```

Comment:
Again, in #403 there is a seeded `BLSPublicKey.random(int)` method which should be used here, I think. @jrhea?
There is already a seeded `BLSSignature.random(int)` method that ought to be used here.

Reply:
I think #403 was merged after v0.4-integration was branched off so leaving it as is until the branch is updated.

Reply:
agreed on using the seed.  @schroedingerscode is going to bring in those changes and review

Reply:
Updated.